packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -265,3 +265,23 @@ describe('getTranslationKey()', () => {
     expect(result).toBe('country');
   });
 });
+
+describe('toSupportedLocale', () => {
+  it('changes the lowercase locale to uppercase', async () => {
+    mockAPICall('country', countryJpJa);
```

Comment:
To make the test more explicit, I would pass explicitly `"ja"` as third argument here.